lesson3/code-examples/examples/exercise.js

```diff
@@ -0,0 +1,172 @@
+// 1. Write function getZodiacSign(day, month) which will return zodiac sign depending on input.
+
+//    Dates for concrete zodiac sign can be found in https://www.horoscopedates.com/zodiac-signs/
+
+function getZodiacSign(day, month) {
+  if (month < 1 || month > 12) {
+    return "Month not available!";
+  }
+  if (
+    (month === 1 && day >= 20 && day <= 31) ||
+    (month === 2 && day >= 1 && day <= 18)
+  ) {
+    return "Aquarius";
+  } else if (
+    (month === 2 && day >= 18 && day <= 29) ||
+    (month === 3 && day >= 1 && day < 20)
+  ) {
+    return "Pisces";
+  } else if (
+    (month === 3 && day >= 20 && day <= 31) ||
+    (month === 4 && day >= 1 && day < 20)
+  ) {
+    return "Aries";
+  } else if (
+    (month === 4 && day >= 20 && day <= 30) ||
+    (month === 5 && day >= 1 && day < 21)
+  ) {
+    return "Taurus";
+  } else if (
+    (month === 5 && day >= 21 && day <= 31) ||
+    (month === 6 && day >= 1 && day < 21)
+  ) {
+    return "Gemini";
+  } else if (
+    (month === 6 && day >= 21 && day <= 30) ||
+    (month === 7 && day >= 1 && day < 23)
+  ) {
+    return "Cancer";
+  } else if (
+    (month === 7 && day >= 23 && day <= 31) ||
+    (month === 8 && day >= 1 && day < 23)
+  ) {
+    return "Leo";
+  } else if (
+    (month === 8 && day >= 23 && day <= 31) ||
+    (month === 9 && day >= 1 && day < 23)
+  ) {
+    return "Virgo";
+  } else if (
+    (month === 9 && day >= 23 && day <= 30) ||
+    (month === 10 && day >= 1 && day < 23)
+  ) {
+    return "Libra";
+  } else if (
+    (month === 10 && day >= 23 && day <= 31) ||
+    (month === 11 && day >= 1 && day < 22)
+  ) {
+    return "Scorpio";
+  } else if (
+    (month === 11 && day >= 22 && day <= 31) ||
+    (month === 12 && day >= 1 && day < 22)
+  ) {
+    return "Sagittarius";
+  } else if (
+    (month === 12 && day >= 22 && day <= 30) ||
+    (month === 1 && day >= 1 && day < 20)
+  ) {
+    return "Capricorn";
+  } else {
+    return "Zodiac sign for given date not exists!";
+  }
+}
+
+// 2. Take data from file users.json and output person names with salary that are bigger than 1000. Try to use *Array* object methods like *forEach*.
+
+function getSalary(url) {
+  return fetch(url)
+    .then(res => {
+      if (!res.ok) {
+        throw new Error(`Erron occured ${res.status}`);
+      }
+      return res.json();
+    })
+    .then(data => {
+      data.forEach(user => {
+        user.salary > 1000 ? console.log(user.name) : "";
```

Comment:
Might be better to `filter` users with salary > 100 into separate array and then output names from it using `foreach`.